### PR TITLE
Update Android integration instructions

### LINKED ISSUE #382: Problems using with Android app
The readme says to add this to gradle:

```
dependencies {
    compile 'io.jsonwebtoken:jjwt-api:0.10.2'
    runtime 'io.jsonwebtoken:jjwt-impl:0.10.2' 
    runtime('io.jsonwebtoken:jjwt-orgjson:0.10.2') {
        exclude group: 'org.json', module: 'json' //provided by Android natively
    }
    // Uncomment the next line if you want to use RSASSA-PSS (PS256, PS384, PS512) algorithms:
    //runtime 'org.bouncycastle:bcprov-jdk15on:1.60'
}
```

However this produces errors as `runtime` is not a valid function in the latest version of gradle.

If I change to `api` or `implementation`, I get proguard failures:

```
Warning: io.jsonwebtoken.io.OrgJsonSerializer: can't find referenced class org.json.JSONString
Warning: io.jsonwebtoken.io.OrgJsonSerializer: can't find referenced class org.json.JSONWriter
Warning: io.jsonwebtoken.io.OrgJsonSerializer: can't find referenced class org.json.JSONWriter
```

### PR DESCRIPTION
**Pull request by tmm1**
fixes #382

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,13 +229,13 @@ Add the dependencies to your project:
 
 ```groovy
 dependencies {
```
Comment (lhazlewood) [inline README.md]: I'm pretty sure this should now be:

```groovy
dependencies {
    api 'io.jsonwebtoken:jjwt-api:0.10.2'
    runtimeOnly 'io.jsonwebtoken:jjwt-impl:0.10.2' 
    runtimeOnly('io.jsonwebtoken:jjwt-orgjson:0.10.2') {
        exclude group: 'org.json', module: 'json' //provided by Android natively
    }
    // Uncomment the next line if you want to use RSASSA-PSS (PS256, PS384, PS512) algorithms:
    //runtimeOnly 'org.bouncycastle:bcprov-jdk15on:1.60'
}
```

and proguard additions should be:

```
-dontwarn org.json.JSONString
-dontwarn org.json.JSONWriter```

Could you please confirm?

Comment (tmm1) [reply README.md]: Yes that works.

**Comment by lhazlewood** [comment]
Also, can you target the 0.10.x branch?  https://github.com/jwtk/jjwt/tree/0.10.x

File: README.md (hunk 1 above)
Comment (lhazlewood) [reply README.md]: Thanks!

Comment (lhazlewood) [reply README.md]: I've updated this in the docs accordingly.  I'll get this out in the 0.10.3 release.  Thanks for your help!

**Comment by tmm1** [comment]
Looks like you committed this already with https://github.com/jwtk/jjwt/commit/4e1d0727571b097e07a6a08bb5cc7bada685d930

**Comment by lhazlewood** [comment]
Yeah, we were working on it at the same time :)

**Pull request closed without merging**